src/languageFeatures.ts

```diff
@@ -326,6 +326,11 @@ export class CompletionAdapter
               documentation: entry.documentation,
               detail: entry.detail,
               kind: toCompletionItemKind(entry.kind),
+              range: {
+                startLineNumber: position.lineNumber,
+                endLineNumber: position.lineNumber,
+                ...wordInfo,
+              },
```

Comment:
`range` is now mandatory. See [Changelog for 0.16.0](https://github.com/microsoft/monaco-editor/blob/master/CHANGELOG.md#0160-05032019)